packages/components/src/components/Task/Task.jsx

```diff
@@ -161,8 +167,9 @@ class Task extends Component {
         data-succeeded={succeeded}
         data-selected={(expanded && !selectedStepId) || undefined}
       >
-        <span
+        <span // eslint-disable-line jsx-a11y/no-static-element-interactions
           className="tkn--task-link"
+          tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
This meant that the Task element wasn't reachable by keyboard, so users couldn't access the Task status, pod info, results, etc. by keyboard. Also couldn't expand other tasks in the tree by keyboard.